### DISCUSSION TITLE
Variable features after adding gene activity to Seurat object

### DISCUSSION THREAD

**Discussion by czfoo** [stuart-lab/signac#1567] [Q&A] [answered]
Hi everyone,

After I ran the GeneActivity function and tried to add it to my Seurat object with the CreateAssayObject function, I noticed that the var.features list is empty. How do I get that list populated? The original gene activities matrix before I performed the CreateAssayObject function has them stored under dimsension 1.

**Comment by timoast** [comment, marked as answer]
https://github.com/stuart-lab/signac/issues/1568
